src/database/migrations/20220321123903-create-accomodation.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable strict */
+
+module.exports = {
+    async up(queryInterface, Sequelize) {
+      await queryInterface.createTable('accomodations', {
```

Comment:
Why did you create new accomodation migration? you can use the already existing one if no major change or delete it.